### ISSUE TITLE
I installed Luma3DS in nand

### ISSUE THREAD

**Issue by bulzipke** [DS-Homebrew/TWiLightMenu#2025]
**Is your feature request related to a problem? Please describe.**
I am a 3DS user, and Luma3DS is installed in NAND.

So I don't have sd:/luma/config.ini
Instead, it is in nand:/rw/luma/config.ini

**Describe the solution you'd like**
Is there a new option so that can select the installation path of the Luma3DS?

**Describe alternatives you've considered**
I think there is also a way to check both sd and nand path, not a new option.

**Comment by RocketRobz** [comment]
I assume you're trying to use the widescreen feature?
If not, then how is this a TWiLight Menu++ issue?

**Comment by bulzipke** [comment]
> I assume you're trying to use the widescreen feature? If not, then how is this a TWiLight Menu++ issue?
That's right. Need a exists check config.ini to use the widescreen feature

**Comment by RocketRobz** [comment]
Unfortunately, it is not possible to implement a NAND check, so that's why TWLMenu++ only checks the file on the SD card.

**Comment by Epicpkmn11** [comment]
Doesn't NAND Luma also check for the TwlBg.cxi on NAND? You can just install SD luma alongside NAND luma, doing what TWiLight does with Widescreen (renaming the file a lot) on NAND would not be good for the console.

**Comment by bulzipke** [comment]
> Doesn't NAND Luma also check for the TwlBg.cxi on NAND? You can just install SD luma alongside NAND luma, doing what TWiLight does with Widescreen (renaming the file a lot) on NAND would not be good for the console.

Looks like something needs to be done as a trick 😄

**Comment by Epicpkmn11** [comment]
Just copy boot.firm to your SD and everything will work properly, Luma on SD has priority

Luma on NAND is only really supposed to be used as a fallback if you boot the console without an SD card

**Comment by bulzipke** [comment]
> Just copy boot.firm to your SD and everything will work properly, Luma on SD has priority
> 
> Luma on NAND is only really supposed to be used as a fallback if you boot the console without an SD card

That's right.
I should consider changing the installation patm from NAND to SD.
